src/containers/category-dropdown/CategoryDropdown.tsx

```diff
@@ -127,13 +125,23 @@ const CategoryDropdown = ({
   )
   return (
     <Box sx={styles.labelCategory}>
-      <Typography variant={TypographyVariantEnum.Body2}>
-        {t('questionPage.chooseCategory')}
-      </Typography>
       <AsyncAutocomplete<CategoryNameInterface>
         fetchCondition={!isFetched}
         fetchOnFocus={isFetchedOnFocus}
         labelField='name'
+        noOptionsText={
+          <AppButton
+            disableRipple
+            fullWidth
+            onClick={onCreateCategory}
+            size={SizeEnum.Medium}
+            sx={styles.addButtonNoOptions}
+            variant={ButtonVariantEnum.Text}
+          >
+            <AddIcon />
+            {t('myResourcesPage.categories.addBtn')}
+          </AppButton>
```

Comment:
Maybe it will be better to move this button above and send a style as a props, and use this there and in the "Options List"